### DISCUSSION TITLE
弹幕文件路径似乎有问题

### DISCUSSION THREAD

**Discussion by Xiefengshang** [libsgh/PanIndex#331] [Q&A] [answered]
RT，我将`字幕路径(path)`设定为`A/`
`弹幕路径(path)`设定为`B/`
重启之后通过抓包发现`a.mp4`视频尝试读取的为`A/a.srt`和`A/a.xml`而非`B/a.xml`.  
但看`Code` `Subtitle`和`danmuku`的处理又是几乎一样的,不清楚是什么问题.

**Comment by libsgh** [comment, marked as answer]
这个是BUG，已经在新版修复，可以试试dev分支构建的版本。

**Reply by Xiefengshang** [reply]
已解决，感谢！
